src/vs/workbench/parts/preferences/browser/settingsEditor2.ts

```diff
@@ -199,13 +205,20 @@ export class SettingsEditor2 extends BaseEditor {
 		previewTextLabel.textContent = localize('previewLabel', "This is a preview of our new settings editor");
 
 		const searchContainer = DOM.append(this.headerContainer, $('.search-container'));
-		this.searchWidget = this._register(this.instantiationService.createInstance(SearchWidget, searchContainer, {
-			ariaLabel: localize('SearchSettings.AriaLabel', "Search settings"),
-			placeholder: localize('SearchSettings.Placeholder', "Search settings"),
-			focusKey: this.searchFocusContextKey,
-			ariaLive: 'assertive'
-		}));
-		this._register(this.searchWidget.onDidChange(() => this.onSearchInputChanged()));
+
+		let searchBoxLabel = localize('SearchSettings.AriaLabel', "Search settings");
+		this.searchWidget = this._register(this.instantiationService.createInstance(AutosuggestEnabledInput, `${SettingsEditor2.ID}.searchbox`, searchContainer, {
+			triggerCharacters: ['@'],
+			provideResults: (query: string) => {
+				return ['@modified', '@tag:usesOnlineServices'].filter(tag => query.indexOf(tag) === -1).map(tag => tag + ' ');
```

Comment:
Without actually trying this, does this mean that if I type `@mod` it will still return `@tag:usesOnlineServices`?

Reply:
Got it, thanks